test/lib/selectors.js

```diff
@@ -7,7 +7,9 @@ module.exports = {
   activeTab: '[data-test-active-tab]',
   activeTabTitle: '[data-test-active-tab] [data-test-id="tabTitle"]',
   activeTabFavicon: '[data-test-active-tab] [data-test-favicon]',
+  pinnedTabs: '.pinnedTabs',
```

Comment:
can we make use of `data` attr instead? something like `data-pinned-tab`. We're trying to avoid tests w/ CSS selectors so Aphrodite changes could less likely break tests

Reply:
sure - I had just copied what was there originally but makes sense to make the positive changes now